p2p/nat/natupnp_test.go

```diff
@@ -29,7 +29,6 @@ import (
 )
 
 func TestUPNP_DDWRT(t *testing.T) {
-	t.Skip("broken")
```

Comment:
This change is not related to removing vendoring....what's the reason to re-enable this test?

Reply:
See comment on PR - these tests are not related to Swarm, and I don't see them failing on go-ethereum master.